### PR TITLE
imp/usage-docs: Add some documentation around installation and gotchas

### PR DESCRIPTION
**Pull request by seesleestak**
Adding some clarification around local installation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,32 @@ const AppContainer = () => (
...
+
+```bash
+npm install -g lerna
+```
+
+To start the documentation React app, run `npm start` inside the `/hui-pages` directory.
+
+## Troubleshooting
+
+Occasionally -- on a fresh install -- `lerna bootstrap` will fail because of a missing `react` dependency. This can be rectified by installing, but not saving the react dependency.
```
Comment (seesleestak) [inline README.md]: I don't know if anyone else has run into this, but I figured I'd document it for good measure.

Comment (billyxs) [reply README.md]: I haven't. good to know

Comment (billyxs) [reply README.md]: Should this go on the HUI contribution docs?

Comment (seesleestak) [reply README.md]: Definitely. Didn't know we had that.

**Pull request merged**